public/doc/openapi-3-v1.2.json

```diff
@@ -1,9 +1,9 @@
 {
     "openapi": "3.0.0",
     "info": {
-        "title": "Insights Service Approval APIs",
-        "description": "APIs to query approval service",
-        "version": "1.2.0",
+        "title": "Insights Approval Service API",
+        "description": "The API to create and query approval requests",
+        "version": "1.2",
```

Comment:
@bzwei This version number directly maps to the gem version that we plan on building. Can we keep the patch version here. Otherwise the gem version number will have to be 1.3. Its better to keep the gem version as 1.2.1 but leave the path version as 1.2